mutate.go

```diff
@@ -90,6 +91,19 @@ func mutate(namespacesLister corev1listers.NamespaceLister, request v1beta1.Admi
 				Operator: v1.TolerationOpEqual,
 				Effect:   v1.TaintEffectNoSchedule,
 			})
+			// put condition here to only add this toleration for oncosim
+			// TODO: you may want another mechanism to identify oncosim which pods get scheduled to the
+			// compute optimized node.
+		} else if request.Namespace == "oncosim" && strings.HasPrefix(pod.Name, "big-cpu") {
```

Comment:
I recommend changing how that condition is triggered:
* Avoid hard-coding the name of the namespace
* Use `requests.cpu` to trigger the toleration rather than a naming convention

To extract the hard-coded namespace, and make it possible to offer this feature to others without redeploying the component, you could use a `ConfigMap` and something like [gokyle/filecache](https://github.com/gokyle/filecache) to compare against a file-based list of `allowedNamespaces` with a refresh period.

Reply:
Potentially a solution would be to target a label or an annotation and just prevent the use of the metadata to specific namespaces similar to what is done with PB pods, currently.